mintapi/signIn.py

```diff
@@ -317,14 +317,20 @@ def sign_in(
     imap_password=None,
     imap_server=None,
     imap_folder="INBOX",
+    beta=False,
 ):
+    if beta:
+        url = constants.MINT_BETA_ROOT_URL
+    else:
+        url = constants.MINT_ROOT_URL
     """
     Takes in a web driver and gets it through the Mint sign in process
     """
     driver.implicitly_wait(20)  # seconds
-    driver.get("https://www.mint.com")
-    element = driver.find_element_by_link_text("Sign in")
-    element.click()
+    driver.get(url)
+    if not beta:
```

Comment:
Referencing the beta URL takes you right to the sign-in page, so this is not needed in that instance.